README.md

```diff
@@ -5,13 +5,18 @@
 ## Synopsis
 
 ```
-phylo2owl [input.(tre|phy|xml|...)] [-o output.owl]
+phylo2owl [input.(tre|phy|xml|...)] [-f format] [-o output.owl]
```

Comment:
I've expanded the command line options section at https://github.com/gaurav/phylo2owl/tree/support_for_nexml_and_nexus#command-line-options -- will that do?